test/player_test.h

```diff
@@ -30,6 +30,16 @@ static int Count8(int8_t *ints, int n)
 	return count;
 }
 
+static int CountU8(uint8_t *ints, int n)
+{
+	int count = n;
+	for (int i = 0; i < n; i++)
+		if (ints[i] == 0)
+			count--;
+
+	return count;
```

Comment:
```suggestion
	return std::count_if(ints, ints + n, [](uint8_t x) { return x != 0; });
```

Reply:
I also updated all other Count*-function to `std::count_if`